e2e-tests/create-cluster_test.go

```diff
@@ -296,9 +312,9 @@ var _ = Describe("CreateCluster", func() {
 		By("generating cluster manifests from example template")
 		cmd := exec.Command(tests.ClusterctlPath, "generate", "cluster", "kvcluster", "--target-namespace", namespace, "--kubernetes-version", "v1.21.0", "--control-plane-machine-count=1", "--worker-machine-count=1", "--from", "templates/cluster-template.yaml")
 		cmd.Env = append(os.Environ(),
-			"NODE_VM_IMAGE_TEMPLATE=quay.io/kubevirtci/fedora-kubeadm:35",
+			"NODE_VM_IMAGE_TEMPLATE=quay.io/isaacdorfman/kubevirt-kubernetes-node",
```

Comment:
Need to change to the official registry before merge

Reply:
Done, but 20.04 instead of 18.04 because that's the image that was used.